tests/pass-dep/libc/libc-mem.rs

```diff
@@ -237,6 +237,32 @@ fn test_reallocarray() {
     }
 }
 
+#[cfg(not(target_os = "windows"))]
+fn test_aligned_alloc() {
+    extern "C" {
+        fn aligned_alloc(_: libc::size_t, _: libc::size_t) -> *mut libc::c_void;
+    }
```

Comment:
Please import the function from `libc` rather than declaring it here.

Reply:
it is not in libc but only for a handful of platforms
https://github.com/search?q=repo%3Arust-lang%2Flibc%20aligned_alloc&type=code

Reply:
Good point, I reported that as an issue: https://github.com/rust-lang/libc/issues/3689